### DISCUSSION TITLE
Which Captcha to use

### DISCUSSION THREAD

**Discussion by TheMorpheus407** [Bootstrap-Academy/Bootstrap-Academy#111] [General]
Okay, folks, so we need to discuss a captcha alternative. 
Currently we're using Google Recaptcha, which might be a problem in terms of GDPR/privacy.
Alternatives:
HCaptcha (which I've heard is not much better)
FriendlyCaptcha (which a user said, they were able to bypass easily)
Turnstile (from Cloudflare, also not much better).
Any ideas or comments?

**Comment by cloudishBenne** [comment]
I did a quick web research and found two potential candidates

- [mosparo](https://mosparo.io/) ([GitHub](https://github.com/mosparo/mosparo))
    nice feature: customizable design
- [ALTCHA](https://altcha.org/) ([GitHub](https://github.com/altcha-org/altcha))
    nice feature: inclusive to robots by providing a captcha for bots, that works as a rate limiter ([more information](https://altcha.org/docs/m2m-altcha))

More alternatives: https://alternativeto.net/software/recaptcha/?license=free
